packages/itwinui-react/src/core/DatePicker/DatePicker.tsx

```diff
@@ -580,27 +632,45 @@ export const DatePicker = React.forwardRef((props, forwardedRef) => {
             </IconButton>
           )}
         </Box>
-        <Box className='iui-calendar-weekdays'>
+        <Box
```

Comment:
What is the difference between `<div>` and `<Box as="div">`?

Reply:
The Box was created for css modules to work. Check Box source code.